bolt-design-system/bolt
# Image-related ratio style bug

## Jira

- http://vjira2:8080/browse/BDS-1239

## Summary

When JS is turned off, some images overflow their container.

## Details

See bug on [master](https://master.boltdesignsystem.com/pattern-lab/patterns/02-components-image-25-image-custom-width-height-variations/02-components-image-25-image-custom-width-height-variations.html) with JS off.

We need to add some CSS to the ratio component to fix this.

## How to test
- Review the following with JS turned off.
- View [this page on master](https://master.boltdesignsystem.com/pattern-lab/patterns/02-components-image-25-image-custom-width-height-variations/02-components-image-25-image-custom-width-height-variations.html) to see bug.
- View [same page on feature branch](https://fix-image-related-ratio-styles.boltdesignsystem.com/pattern-lab/patterns/02-components-image-25-image-custom-width-height-variations/02-components-image-25-image-custom-width-height-variations.html) to see fix.
- Review [CSS change](https://github.com/bolt-design-system/bolt/pull/1156/files).
